insights/parsers/crictl_logs.py

```diff
@@ -0,0 +1,30 @@
+"""
+CrictlLogs - files ``/usr/bin/crictl logs -t <list_of_containers'_IDs>``
```

Comment:
`And I think it should be Commands but not files.`

You missed this comment.

Reply:
Thanks! Updated it